Alcancia-io/alcancia
# Feat/onboarding screens

## Summary
Adds onboarding screens for first app run

## Requirements
N/A

## Type of change

## How to test?
Run the app, the onboarding screens should appear. Tap skip or register buttons and reset the app (either hot restart or closing and reopening), onboarding screens should not show up again.
## How has this been tested?
- [ ] Tap skip button and reopen app
- [ ] Tap register button and reopen app
- [ ] Delete app and reinstall, onboarding should show up again

## Screenshots
_add screenshots if need it_

## Checklist (for the reviewer)
